src/cli.py

```diff
@@ -19,21 +19,86 @@
 
 '''  to run > sudo LD_LIBRARY_PATH=../libcsp/build PYTHONPATH=../libcsp/build python3 src/cli.py -I uart -d /dev/ttyUSB1  '''
 import time
+import libcsp_py3 as libcsp
+import unicodedata
 from groundStation import groundStation
 
 opts = groundStation.options()
 gs = groundStation.groundStation(opts.getOptions())
 flag = groundStation.GracefulExiter()
 
 def cli():
+    #csp = groundStation(opts.getOptions())
+    sysVals = groundStation.SystemValues()
+
+    #strVal = u'12 * 13 14 2 2 52'
+    #print(unicodedata.normalize('NFKD', strVal).encode('ascii', 'replace').decode())
+
     while True:
         if flag.exit():
-            print('Exiting receiving loop')
+            print('Exiting receiving loop\n')
             flag.reset()
             return
         try:
             server, port, toSend = gs.getInput(prompt='to send: ')
-            resp = gs.transaction(server, port, toSend)
```

Comment:
I am a bit confused why we're calling getInput and transaction here in cli.py and again in groundStation.py?